driver/InputFlags.cpp

```diff
@@ -146,6 +161,8 @@ void InputFlags::Parse(int argc, char* argv[])
             else
             {
                 MapInputs[short_name].value = args[i + 1];
+
+                std::cout << "Input Flag:::else:value of short name " << MapInputs[short_name].value << std::endl;
```

Comment:
Is this a debug print? Perhaps should clean up.

Reply:
Ok. I will check and update the code. Thank you. @cderb